data/virt_autotest/kubevirt_tests_config_parser.pl

```diff
@@ -0,0 +1,21 @@
+use strict;
+use warnings;
+use Config::Tiny;
+
+my $config    = Config::Tiny->read($ARGV[0]);
+my $go_test   = '';
+my $skip_test = '';
+my $extra_opt = '';
+
+foreach my $key (keys %{$config->{$ARGV[1]}}) {
+    if ($key eq 'GINKGO_FOCUS') {
+        $go_test = $config->{$ARGV[1]}->{$key};
+    } elsif ($key eq 'GINKGO_SKIP') {
+        $skip_test = "|$config->{$ARGV[1]}->{$key}";
+    } elsif ($key eq 'EXTRA_OPT') {
+        $extra_opt = $config->{$ARGV[1]}->{$key};
+    }
+}
+
```

Comment:
There is also SOFT_FAIL key in kubevirt-tests.conf, but there is no corresponding handling in this config parser.

Reply:
And this config parser can not get all keys' values from kubevirt-tests.conf, because if and elsif branches are exclusive.

Reply:
Yes, this SOFT_FAIL belongs to vmi_configuration_test which is disabled in the whole tests, and it will be manually executed in this roadmap. In next stage, we will add related handling for SOFT_FAIL and enable its test.